rnc2rng/parser.py

```diff
@@ -70,6 +70,12 @@ def __repr__(self):
     'SEQ', 'SOME', 'TEXT',
 ]
 
+for _name in NODE_TYPES:
+    assert _name not in globals()
+    globals()[_name] = _name
+
+del _name
```

Comment:
I think the `del` here is overkill.

`_name` is kind of ugly to me, I would probably use `nt` or something.